.changeset/late-candles-work.md

```diff
@@ -0,0 +1,7 @@
+---
+"@yamada-ui/for": minor
```

Comment:
Please remove this line.

If I am publishing a package to npm for the first time, I must manually publish it.

Reply:
Removed.